tests/datalab/test_cleanvision_integration.py

```diff
@@ -90,7 +90,7 @@ def test_imagelab_issues_checked(
                     "outlier",
                     "near_duplicate",
                 ],
-                "num_issues": [1, 1, 2, 1, 1, 1, 1, 0, 0, 0],
+                "num_issues": [1, 1, 0, 1, 1, 1, 1, 0, 0, 0],
```

Comment:
open a "help-wanted" GH issue to make this test more comprehensive with a dataset that sometimes has more than 1 instance of certain issues